integrated_channels/integrated_channel/tasks.py

```diff
@@ -90,6 +91,45 @@ def _log_batch_task_finish(task_name, channel_code, job_user_id,
         ))
 
 
+@shared_task
+@set_code_owner_attribute
+def remove_duplicate_transmission_audits():
+    """
+    Task to mark content metadata audits as orphaned if they are not linked to any customer catalogs.
+    """
+    start = time.time()
+    _log_batch_task_start('remove_duplicate_transsmision_audits', None, None, None)
+    unique_transmissions = ContentMetadataItemTransmission.objects.values_list(
+        'content_id',
+        'plugin_configuration_id',
+        'integrated_channel_code',
+    ).distinct()
+    duplicates_found = 0
+    for unique_transmission in unique_transmissions:
+        duplicates = ContentMetadataItemTransmission.objects.filter(
+            id__in=ContentMetadataItemTransmission.objects.filter(
+                content_id=unique_transmission[0],
+                plugin_configuration_id=unique_transmission[1],
+                integrated_channel_code=unique_transmission[2]
+            ).order_by('-modified').values_list('id', flat=True)[1:]
+        )
+        duplicates_found += len(duplicates)
+        if getattr(settings, "REMOVE_DUPLICATE_CONTENT_TRASMISSION_FLAG", False):
```

Comment:
Do we want to do all of the above work if the flag is not set/true?  Is this intended to be a "dry run" flag?

Reply:
If the flag exists and is set to true, then we want it to delete. Otherwise log and dry run

Reply:
Okay, if it's a "dry run" control, I recommend naming the parameter as such -- otherwise people may think the flag is an on/off switch for the entire workflow which isn't entirely true.

Reply:
basically we want to see exactly how many records the code intends to delete before we issue the operation